doofinder.php

```diff
@@ -260,7 +260,8 @@ public function getContent()
             return $stop;
         }
         $this->migrateOldConfigHashIDs();
-        $adv = Tools::getValue('adv', 0);
+        $adv = Tools::getValue('adv', 1);
```

Comment:
why do you want to show the "advanced" tab always?

Reply:
Sorry, I had to add this manually because adding the parameter to the url does not work in my local environment and I forgot removing it back, thanks for pointing it.